scripts/gh-projects/examples/matchline/phase-3/c1.md

```
@@ -0,0 +1,16 @@
+Improve the extraction prompt against Nathan's real resumes.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)